compile.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+usage() {
+    echo '                Please export USE_TESTENV_PROPERTIES, JDK_VERSION and JDK_IMPL before running the script locally.'
+
+}
+if [ $(uname) = AIX ] || [ $(uname) = SunOS ] || [ $(uname) = *BSD ]; then
+    MAKE=gmake
+else
+    MAKE=make
+fi
+if [ $USE_TESTENV_PROPERTIES == true ]; then
+    while read line; do
+        export $line
+    done <./testenv/testenv.properties
+    repo_j9=JDK${JDK_VERSION}_OPENJ9_REPO
```

Comment:
can we move line 15 - 22 to the following if [ $JDK_IMPL == "openj9" ] block or else block depends on properties?

something like:

```
if [ $JDK_IMPL == "openj9" ]; then
     repo=JDK${JDK_VERSION}_OPENJ9_REPO
     branch=JDK${JDK_VERSION}_OPENJ9_BRANCH
else
     repo=JDK${JDK_VERSION}_REPO
     branch=JDK${JDK_VERSION}_BRANCH
fi

    eval repo2='$'$repo
    eval branch2='$'$branch

    export JDK_REPO=$repo2
    export JDK_BRANCH=$branch2
```